### PR TITLE
[CDAP-17371] Add error count metric at the table level.

### PR DESCRIPTION
**Pull request by sagarkapare**
Also made other metric name plural. For example dml.insert->dml.inserts.

### CODE REVIEW

File: delta-app/src/main/java/io/cdap/delta/app/EventMetrics.java [hunk 1]
```diff
@@ -52,12 +53,17 @@ public synchronized void incrementDDLCount() {
...
   }
 
+  public synchronized void incrementDMLErrorCount() {
+    dmlErrorCount++;
+  }
+
   public synchronized void emitMetrics() {
     for (DMLOperation.Type op : dmlEventCounts.keySet()) {
-      metrics.count(String.format("dml.%s", op.name().toLowerCase()), dmlEventCounts.get(op));
+      metrics.count(String.format("dml.%ss", op.name().toLowerCase()), dmlEventCounts.get(op));
```
Comment (seanzhougoogle) [inline delta-app/src/main/java/io/cdap/delta/app/EventMetrics.java:62]: does change of metrics name have any impact on UI side ?

**Review by seanzhougoogle — APPROVED** [review]
Thank you Sagar :)

File: delta-app/src/main/java/io/cdap/delta/app/EventMetrics.java (hunk 1 above)
Comment (sagarkapare) [reply delta-app/src/main/java/io/cdap/delta/app/EventMetrics.java:62]: yes. Have talked to @elfenheart about it. Those changes will be done once the PR is merged.

**Comment by sagarkapare** [comment]
https://builds.cask.co/browse/DEL-BT92 Build is green. Squashing and merging.

**Pull request merged**